src/scripts/match-mondo-sources-all-lexical.py

```diff
@@ -108,19 +112,17 @@ def run(input: str, config: str, rules: str, rejects: str, output: str):
     #     .reset_index(drop=True)
     # )
 
-    prefix_of_interest = yml["subject_prefixes"]
-
-    resource = OntologyResource(slug=f"sqlite:///{Path(input).absolute()}")
+    resource = OntologyResource(slug=f"sqlite:///{Path(_input).absolute()}")
     oi = SqlImplementation(resource=resource)
     ruleset = load_mapping_rules(rules)
     # syn_rules = [x.synonymizer for x in ruleset.rules if x.synonymizer]
     lexical_index = create_lexical_index(oi=oi, mapping_rule_collection=ruleset)
     save_lexical_index(lexical_index, OUT_INDEX_DB)
 
     if rules:
-        msdf = lexical_index_to_sssom(oi, lexical_index, ruleset=ruleset, meta=meta)
+        msdf = lexical_index_to_sssom(oi, lexical_index, ruleset=ruleset, prefix_map=converter)
```

Comment:
## `meta` -> `converter`
Along with removing `oio` from `config/prefixes.csv`, this is the essential bugfix. This allows us to use the new extended prefix map, and solves the `.pop()` error in the original issue #390.